docker/docker-compose.override.yml.default

```diff
@@ -10,6 +10,8 @@ services:
       - "./90-mail.ini:/etc/php7/conf.d/90-mail.ini:z"
 #    depends_on:
 #      - mysql
+# Catch code assertions "development" mode. https://www.php.net/manual/en/function.assert.php
+     command: php-fpm7 -F -dzend.assertions=1
```

Comment:
missing space after `-d`

Reply:
The space is optional here, otoh I think it could use .env override for command because it should not bleed to live sites

Reply:
dev images could use this settings too but as this override is copied to every custom project for local development we need that here